### PR TITLE
Multi label margin loss

### PR DESCRIPTION
**Pull request by v0dro**
Reopen PR for https://github.com/pytorch/pytorch/pull/46975

### CODE REVIEW

**Comment by v0dro** [comment]
@ngimel I've reopened the PR for Multi label margin loss following failure for asan. Could you please review it again?

**Review by ngimel — APPROVED** [review]

**Comment by v0dro** [comment]
@rgommers any update on when this can be merged?

**Comment by rgommers** [comment]
> @rgommers any update on when this can be merged?

It's been imported, so should be relatively soon unless there's FB-internal CI failures. That said, there's a lint failure you still need to address it looks like @v0dro. Can you please do that?

File: aten/src/ATen/native/LossMultiLabelMargin.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #include <ATen/AccumulateType.h>
 #include <ATen/Dispatch.h>
 #include <ATen/TensorUtils.h>
+#include "LossMulti.h"
```
Comment (ngimel) [inline aten/src/ATen/native/LossMultiLabelMargin.cpp:5]: please use standard include here, this leads to broken builds.

**Comment by ngimel** [comment]
Sorry, there's an internal build failure because of non-standard include, I thought I left the comment about it but apparently I didn't. Please fix the include.

File: aten/src/ATen/native/LossMultiLabelMargin.cpp (hunk 1 above)
Comment (v0dro) [reply aten/src/ATen/native/LossMultiLabelMargin.cpp]: My apologies I missed the comment in the previous PR. Have fixed it now and waiting for the build results.

**Comment by ngimel** [comment]
Can you please rebase to get rid of mypy errors?

**Pull request closed without merging**